### DISCUSSION TITLE
Upgrade from Win10 to Win11

### DISCUSSION THREAD

**Discussion by parapebune** [AveYo/MediaCreationTool.bat#336] [Q&A]
Hi! First of all, thanks for your amazing work! Second, I used MediaCreationTool.bat to upgrade from Win10 to Win11, I have chosen version 23H2 from the menu and it upgraded to Home Edition, Single User. How can I upgrade to Win 11 Pro or Enterprise? Sorry if it has already been discussed, I did not find anything on the subj

**Comment by nondetect** [comment]
This tool update only in same edition, if have Home it's update to Home, if Pro -> Pro
In your case I think you have laptop(if not it's doesn't matter) with pre-installed license of Home Single Languageso you need to buy Pro key and upgrade your license to Pro, how to do that just Google it
